src/user/user.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Column, Entity, PrimaryColumn, PrimaryGeneratedColumn } from 'typeorm';
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+  @Column({ unique: true })
+  email: string;
+  @Column()
+  password: string;
+  @Column()
+  firstName: string;
+  @Column()
+  lastName: string;
+  @Column()
+  avatar: string;
```

Comment:
Add empty lines

Reply:
Use cammelcase for variable name but snakecase for column name

Reply:
done